openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -0,0 +1,196 @@
+# Azure Landing Zones paths
+paths:
+  /api/azure/landingzones/v1:
+    post:
+      summary: Create Azure landing zone
+      operationId: createAzureLandingZone
+      tags: [ LandingZones ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateAzureLandingZoneRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/CreateAzureLandingZoneResponse'
+        '202':
+          $ref: '#/components/responses/CreateAzureLandingZoneResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/azure/landingzones/v1/create-result/{jobId}:
+    parameters:
+      - $ref: '#/components/parameters/JobId'
+    get:
+      summary: Get the result of a async job to create an Azure Landing Zone
+      operationId: getCreateAzureLandingZoneResult
+      tags: [ LandingZones ]
+      responses:
+        '200':
+          $ref: '#/components/responses/CreateAzureLandingZoneResponse'
+        '202':
+          $ref: '#/components/responses/CreateAzureLandingZoneResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/azure/landingzones/definitions/v1:
+    get:
+      summary: List all Azure landing zones definitions
+      operationId: listAzureLandingZonesDefinitions
+      tags: [ LandingZones ]
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AzureLandingZoneDefinitionList'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/azure/landingzones/v1/{landingZoneId}:

```

Comment:
+1, maybe something like:
```
# create
POST /api/landingzones/v1

# get create-result
GET /api/landingzones/v1/create-result/{jobId}:

# list
GET /api/landingzones/v1

# get
GET /api/landingzones/v1/{landingZoneId}

# delete
DELETE /api/landingzones/v1/{landingZoneId}

# update
PATCH /api/landingzones/v1/{landingZoneId}
```

Reply:
That's probably will be added later. But for now, there are only 3 operations available - 1) List all landing zone definitions 2) Create landing zone (this is represented by 2 endpoints because of async nature of create operation) 3) Delete landing zone.